CONTRIBUTING.md

```diff
@@ -88,7 +88,7 @@ To run the SDK on an Apple ARM chip, upgrade your Python version to 3.8 or highe
 The set up is almost identical as above except one command; to create a virtual environment for Python 3.8 you should run:
```

Comment:
remove reference to `python 3.8`. Also, are the new changes going to work on ARM chip? I think M1/2s may be using ARM. Should we ask anyone using them to double check?